src/app/camping-vliegenbos/navigationMenuItems.ts

```diff
@@ -0,0 +1,38 @@
+const navigationMenuItems = [
+  {
+    heading: 'Algemeen',
+    items: [
+      {
+        href: '/camping-vliegenbos/over-de-camping',
+        label: 'Over de camping',
+      },
+      {
+        href: '/camping-vliegenbos/zoek-en-boek',
+        label: 'Zoek en boek',
+      },
+      {
+        href: '/camping-vliegenbos/route',
+        label: 'Route',
+      },
+    ],
+  },
+  {
+    heading: 'Accommodaties',
+    items: [
+      {
+        href: '/camping-vliegenbos/tenten',
+        label: 'Tenten',
+      },
+      {
+        href: '/camping-vliegenbos/campers-en-caravans',
+        label: 'Campers en caravans',
+      },
+      {
+        href: '/camping-vliegenbos/trekkershutten',
+        label: 'Trekkershutten',
+      },
+    ],
+  },
+]
+
+export default navigationMenuItems
```

Comment:
This is not a rule or anything, but I generally avoid default exports. A default export means you can import something like this:

`import navigationMenuItems from './notNavigationMenuItems'`

Which you can't catch programmatically, and is very hard to find when it goes wrong.